config/schedule.yml

```diff
@@ -1,113 +1,139 @@
 feeds_import:
+  description: "Imports feed items as articles (runs hourly on the 20th minute after the hour)"
   cron: "20 * * * *" # every hour, 20 min after the hour
   class: "Feeds::ImportArticlesWorker"
 log_worker_queue_stats:
+  description: "Records Sidekiq stats in Datadog (runs every 10 minutes)"
   cron: "*/10 * * * *" # every 10 minutes
   class: "Metrics::RecordBackgroundQueueStatsWorker"
 record_daily_usage:
+  description: "Records daily usage stats (runs daily at 11:00 UTC)"
   cron: "0 11 * * *" # daily at 11:00 UTC
   class: "Metrics::RecordDailyUsageWorker"
 record_daily_notifications:
+  description: "Records daily notifications stats (runs daily at 11:00 UTC)"
   cron: "0 11 * * *" # daily at 11:00 UTC
   class: "Metrics::RecordDailyNotificationsWorker"
 record_data_counts:
+  description: "Records the size of the most import database tables (runs hourly on the 10th minute after the hour)"
```

Comment:
@rhymes this description doesn't make sense, is it `important database`?

Reply:
"record the size of the most important tables in the database" - this what I meant :D